litmus-portal/backend/subscriber/subscriber.go

```diff
@@ -0,0 +1,99 @@
+/*
+	----------------------Subscriber Logic---------------------------------
+	Step1: Check if cluster is already registered or not from the configmap.
+	Step2: Register the subscriber with the admin cluster.
+	Step3: Waiting for request from the graphql server using subscription.
+	NOTE: format:
+	{
+		"requestid": Int!, // Randomly Generated id from the server.
+		"requesttype": String!, // request type can be GET, UPDATE, CREATE and DELETE.
+		"manifest": Object, // K8s manifest, Not needed when it is a GET or DELETE request.
+		"resourcetype": String!, // It can be pre defined or custom k8s resource.
+		"namespace": "default" // namespace of the k8s resource.
+	}
+	! stands for required fields.
+	Step4: Call the the clusterOperation function using the above format.
+	Step5: Response back to the graphql Server.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"github.com/litmuschaos/litmus/litmus-portal/backend/subscriber/pkg/cluster"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+)
+
+var GRAPHQL_SERVER_ADDRESS = os.Getenv("SERVER_ADDRESS") // Format: http://IP:PORT/query
+
+var kubeconfig *string
+
+func init() {
+	if GRAPHQL_SERVER_ADDRESS == "" {
+		panic("GRAPHQL SERVER ADDRESS NOT FOUND")
```

Comment:
Use of panic is necessary to make the pod into crashbackloop if the address is not present.